ee/apps/queue-worker/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+	"extends": "../../../tsconfig.base.server.json",
+	"compilerOptions": {
```

Comment:
ideally you should not need additional `compilerOptions`, at least not this many

Reply:
Oh, this was templated from account-service 👀

Reply:
probably before the changes then.. so you're probably good to remove almost everything ;)